Description: Correct bindings for use with VTK6 
 Add VTK6 export defines and remove some vtk-python libraries
 that no longer exist from the link command 
Author: Gert Wollny <gw.fossdev@gmail.com>

diff -ru gdcm-2.6.0.old/Utilities/VTK/CMakeLists.txt gdcm-2.6.0/Utilities/VTK/CMakeLists.txt
--- gdcm-2.6.0.old/Utilities/VTK/CMakeLists.txt	2015-09-03 16:42:12.000000000 +0200
+++ gdcm-2.6.0/Utilities/VTK/CMakeLists.txt	2015-10-16 17:28:48.000000000 +0200
@@ -740,9 +740,9 @@
     # do not set the version on the Python module:
     #set_target_properties(${VTKGDCM_NAME}Python PROPERTIES ${GDCM_LIBRARY_PROPERTIES})
     target_link_libraries(${VTKGDCM_NAME}PythonD ${VTKGDCM_NAME} ${PYTHON_LIBRARY})
-    foreach(c ${vtkgdcm_LIBS} vtkFiltering)
-      target_link_libraries(${VTKGDCM_NAME}PythonD ${c}Python${DEXTENSION})
-    endforeach()
+#    foreach(c ${vtkgdcm_LIBS} vtkFiltering)
+#      target_link_libraries(${VTKGDCM_NAME}PythonD ${c}Python${DEXTENSION})
+#    endforeach()
     if(TARGET vtkPythonCore)
       target_link_libraries(${VTKGDCM_NAME}PythonD vtkPythonCore)
     endif()
diff -ru gdcm-2.6.0.old/Utilities/VTK/vtkgdcm.i gdcm-2.6.0/Utilities/VTK/vtkgdcm.i
--- gdcm-2.6.0.old/Utilities/VTK/vtkgdcm.i	2015-09-03 16:42:12.000000000 +0200
+++ gdcm-2.6.0/Utilities/VTK/vtkgdcm.i	2015-10-16 17:42:20.541502614 +0200
@@ -31,10 +31,10 @@
 #endif
 
 %{
-//#define VTK_MAJOR_VERSION 5
-//#define VTK_MINOR_VERSION 4
+//#define VTK_MAJOR_VERSION 6
+//#define VTK_MINOR_VERSION 2
 //#define VTK_BUILD_VERSION 0
-//#define VTK_VERSION "5.4.0"
+//#define VTK_VERSION "6.2.0"
 %}
 
 
@@ -101,17 +101,24 @@
 //using My.Own.Namespace;
 //%}
 
-#ifdef USEACTIVIZ
+#if defined( USEACTIVIZ ) || defined (SWIGCSHARP) 
 %typemap(csimports) SWIGTYPE %{
 // I need to duplicate those also:
 using System;
 using System.Runtime.InteropServices;
+%}
+#endif
+
+#if defined( USEACTIVIZ )
+%typemap(csimports) SWIGTYPE %{
 // my special import:
 using Kitware.VTK;
 //using Kitware.mummy.Runtime;
 %}
 #endif
 
+
+
 #define GDCM_EXPORT
 #define VTK_EXPORT
 #define VTK_COMMON_EXPORT
@@ -119,6 +126,13 @@
 #define VTK_IO_EXPORT
 #define VTK_IMAGING_EXPORT
 #define VTK_RENDERING_EXPORT
+#define VTKCOMMONCORE_EXPORT
+#define VTKCOMMONMATH_EXPORT
+#define VTKIOIMAGE_EXPORT
+#define VTKCOMMONDATAMODEL_EXPORT
+#define VTKCOMMONEXECUTIONMODEL_EXPORT
+#define VTKRENDERINGCORE_EXPORT
+#define VTKIMAGINGCORE_EXPORT
 
 
 // FIXME. Including #include vtkSetGet would not work on siwg 1.3.33 ...
